.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Seems like the checks are no longer running on new pushes. Might be necessary to add this:

```suggestion
  pull_request:
    branches: [ main ]
    types: [ opened, synchronize ]
```

Reply:
My bad, the merge conflict is what is preventing checks from being run.